# MPC formulation setting
dt: 0.05
N: 20

# MAV dynamics param
mass: 1.56
thrust_scale: 20
roll_time_constant: 0.3256
roll_gain: 1.0234
pitch_time_constant: 0.3249
pitch_gain: 1.0266
drag_coefficient_x: 0.001
drag_coefficient_y: 0.001

# dynamic config default values
q_x : 80
q_y : 80
q_z : 120

q_vx : 80
q_vy : 80
q_vz : 100

r_roll  : 50
r_pitch : 50
r_thrust : 1

roll_max: 25.0        # deg
pitch_max: 25.0       # deg
thrust_min: 0.5       # m*g
thrust_max: 1       # m*g

K_yaw: 1.8
yawrate_max: 90.0     # deg

